diagnostics-app/src/app/events/events-dashboard/events-dashboard.component.ts

```diff
@@ -45,6 +45,7 @@ export class EventsDashboardComponent implements OnInit {
       statusInfo = statusInfo as EventSupportStatusInfo;
```

Comment:
Should we update `_visibleCategories` in this file to use `VISIBLE_EVENT_CARDS`?

Reply:
Done.